cmd/run.go

```diff
@@ -24,6 +28,11 @@ import (
 	"go.temporal.io/sdk/log"
 )
 
+const (
+	summaryListenAddr = "127.7.2.1:0"
+	FeaturePassed     = "PASSED"
```

Comment:
Do we need to define this twice?

Reply:
I thought about introducing a separate module defining the protocol that both the harness and the cmd modules could depend on but it seemed like a too much complexity just to share some constants. Absent that, it seems reasonable for both the producer (the lang-specific runner) and the consumer (the CLI driver) to have this constant. Java, TS, Python, etc. will have it eventually as well.